From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 25 Oct 2019 15:45:38 -0400
Subject: [PATCH] Add stuff that's got weird characters to .gitattributes

Some of these files will be removed in a later patch, but they need to
stay in .gitattributes until after the next release, because they make
patch application break in weird ways.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 .gitattributes | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 .gitattributes

diff --git a/.gitattributes b/.gitattributes
new file mode 100644
index 00000000000..b9c1467125b
--- /dev/null
+++ b/.gitattributes
@@ -0,0 +1,7 @@
+inc/inc.mak binary
+inc/make.inf binary
+inc/makefile.hdr binary
+inc/protocol/efidbg.h binary
+inc/protocol/ia64/eficontext.h binary
+inc/protocol/make.inf binary
+inc/protocol/makefile.hdr binary
